### PR TITLE
restored changes lost when merged storage-ng patch

### PR DESCRIPTION
**Pull request by mchf**
Some updates got lost when https://github.com/yast/yast-network/pull/553 was merged.

### CODE REVIEW

**Comment by mchf** [comment]
@ancorgs Could you pls check if everything needed from storage-ng is here?

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Aug 17 19:42:51 UTC 2017 - mfilka@suse.com
+
+- reverted part of version 3.3.3 which was accidentaly dropped in
```
Comment (mvidner) [inline package/yast2-network.changes]: s/reverted/restored/

**Comment by mvidner** [comment]
I'm just wondering how we found this. Did a test break?
The changed code looks covered by tests that pass before *and* after this fix??

**Comment by mchf** [comment]
@mvidner I found it when I was working on another issue and missed some new calls.

**Comment by mchf** [comment]
thanks for the reviews.

**Pull request merged**